pom.xml

```diff
@@ -61,16 +61,13 @@
     <dep.httpcore.version>5.2.1</dep.httpcore.version>
     <dep.impsort.version>1.8.0</dep.impsort.version>
     <dep.jackson.version>2.13.4</dep.jackson.version>
-    <dep.jakarta.activation-api.version>1.2.2</dep.jakarta.activation-api.version>
```

Comment:
This would be 2.0.1 if it was specified, but removing the override and allowing `jakarta.xml.bind-api` to bring in its own seemed fine.